src/components/accordion-search.tsx

```diff
@@ -104,6 +110,7 @@ const AccordionSearchCheckbox = ({
         checked={selected.includes(id)}
       />
       <SubstringHighlight substring={query}>{label}</SubstringHighlight>
+      {selected.includes(id) && addAsterix ? '*' : ''}
```

Comment:
```suggestion
      {addAsterix && selected.includes(id) ? '*' : ''}
```
A minor suggestion. Possibly a little faster to have this order as `addAsterix` will be a very quick quick and so if false it won't bother to look at `selected.includes(id)`